python/pyproject.toml

```diff
@@ -2,6 +2,39 @@
 requires = ["maturin==0.11.2"]
 build-backend = "maturin"
 
+[project]
+name = "deltalake"
+requires-python = ">=3.6"
+classifier = [
```

Comment:
This should be `classifiers`.

Reply:
good catch @messense , do you want to submit a PR?